xarray/core/dataset.py

```diff
@@ -2285,10 +2287,88 @@ def rename(self, name_dict=None, inplace=None, **names):
                                  "variable or dimension in this dataset" % k)
 
         variables, coord_names, dims, indexes = self._rename_all(
-            name_dict=name_dict, dim_dict=name_dict)
+            name_dict=name_dict)
         return self._replace(variables, coord_names, dims=dims,
                              indexes=indexes, inplace=inplace)
 
+
+    def rename_dims(self, name_dict=None, inplace=None, **names):
+        """Returns a new object with renamed dimensions only.
+
+        Parameters
+        ----------
+        name_dict : dict-like, optional
+            Dictionary whose keys are current dimension names and
+            whose values are the desired names.
+        inplace : bool, optional
+            If True, rename imensions in-place. Otherwise,
+            return a new dataset object.
+        **names, optional
+            Keyword form of ``name_dict``.
+            One of name_dict or names must be provided.
+
+        Returns
+        -------
+        renamed : Dataset
+            Dataset with renamed dimensions.
+
+        See Also
+        --------
+        Dataset.swap_dims
+        Dataset.rename
+        Dataset.rename_vars
+        DataArray.rename
+        """
+        inplace = _check_inplace(inplace)
+        name_dict = either_dict_or_kwargs(name_dict, names, 'rename')
+        for k, v in name_dict.items():
+            if k not in self.dims:
+                raise ValueError("cannot rename %r because it is not a "
+                                 "dimension in this dataset" % k)
+
+        dims = self._rename_dims(name_dict)
+        indexes = self._rename_indexes(name_dict)
+        return self._replace(dims=dims, indexes=indexes, inplace=inplace)
+
+
+    def rename_vars(self, name_dict=None, inplace=None, **names):
```

Comment:
What do people think about `vars` vs `variables` here?

I'm +0.2 on `variables` for consistency with `merge_variables` etc (but only did a quick search, I would update if there were more public `vars` methods around)